### PR TITLE
Store private Docker registry images on Cloud Files

### LINKED ISSUE #622: Store private Docker registry images on Cloud Files tutorial
Adapt this blog post [Flexible Private Docker Registry Infrastructure](http://tech.paulcz.net/2016/01/flexible-docker-registry-infrastructure/) into a tutorial and give a shoutout to the original author in the Resources section.

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #622

### CODE REVIEW

**Comment by kmsholcomb** [comment]
Hi @everett-toews. My suggested edits are in the attached Word doc. Please let me know if you have any questions. Thanks!

[Store private Docker registry images on Rackspace Cloud Files_editsKH.docx](https://github.com/getcarina/getcarina.com/files/89526/Store.private.Docker.registry.images.on.Rackspace.Cloud.Files_editsKH.docx)

**Comment by everett-toews** [comment]
@kmsholcomb Can you give this one last look? 

If you're okay with it, go ahead and merge.

If you've got some comments, I can fix them up tomorrow.

Thanks!

File: _tutorials/2016-01-13-registry-on-cloud-files.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+1. Export environment variables with your Rackspace information.
+
+    ```
+    $ export RS_USERNAME=your-rackspace-cloud-username
+    $ export RS_PASSWORD=your-rackspace-cloud-password
+    $ export RS_REGION=IAD
+    ```
+
+1. Run the Docker registry service
```
Comment (kmsholcomb) [inline _tutorials/2016-01-13-registry-on-cloud-files.md]: End the sentence with a period.

**Comment by kmsholcomb** [comment]
Hey @everett-toews: That section is much clearer now. Thanks! I had just one edit; I left a comment in the file. :+1:

**Pull request merged**